contracts/interfaces/IRibbonVault.sol

```diff
@@ -0,0 +1,16 @@
+// SPDX-License-Identifier: MIT
+pragma solidity =0.8.4;
+
+import {Vault} from "../libraries/Vault.sol";
+
+interface IRibbonVault {
```

Comment:
We could add IRibbonVault into IRibbon.sol instead to centralize the interfaces

Reply:
We have another file `IRibbonThetaVault.sol`.  Do we move that as well or should I delete it?

Reply:
Done